AUTHORS

```diff
@@ -0,0 +1,8 @@
+# Below is a list of people and organizations that have contributed
+# to the Flutter project. Names should be added to the list like so:
+#
+#   Name/Organization <email address>
+
+Google Inc.
```

Comment:
No problem to ask. :)  My understanding is that all the Googlers who have contributed to this project have technically attributed their copyrights to Google, so yes.  When I contribute under this account I'm contributing on behalf of Google, Inc. is my understanding.